int16 base_motor
int16 link_1
int16 link_2
int16 link_3
int16 waist
int16 claw